R/declare_diagnosands.R

```diff
@@ -192,20 +192,53 @@ validation_fn(diagnosand_handler) <- function(ret, dots, label) {
 #'
 #' my_diagnosands <- declare_diagnosands(select = c(bias, rmse))
 #'
+#' \dontrun{
+#' diagnosis <- diagnose_design(design, diagnosands = my_diagnosands)
+#' diagnosis
```

Comment:
I would try to match the structure between the two examples you added - either create a  diagnosis variable in both, or have `diagnose_design` go straight to terminal in both.